dev/PowerNotifications/PowerNotifications.cpp

```diff
@@ -0,0 +1,501 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <PowerNotifications.h>
+#include <PowerNotificationsPal.h>
+#include <PowerManager.g.cpp>
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+    
+    PowerManager::eventType PowerManager::m_energySaverStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_batteryStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_powerSupplyStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_remainingChargePercentChangedEvent;
+    PowerManager::eventType PowerManager::m_remainingDischargeTimeChangedEvent;
+
+    PowerManager::eventType PowerManager::m_powerSourceStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_displayStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_systemIdleStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_powerSchemePersonalityChangedEvent;
+    PowerManager::eventType PowerManager::m_userPresenceStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_systemAwayModeStatusChangedEvent;
+
+    EnergySaverStatusRegistration      PowerManager::m_energySaverStatusHandle;
```

Comment:
While the CppCoreGuidelines don't take a position on this, please don't use "vertical alignment" - it makes maintaining the codebase much harder over time.  @chrisguzak was looking into adding a .editorrc to get consistency for new code.

Reply:
I'm curious : how does it make it harder? It's just to make it look better

Reply:
The problem is maintenance.  When you add a new member here of a type whose name is _longer_ you have to go shift the layout here again, which makes reviewing code changes harder.  Please eliminate these alignment spaces.

Reply:
Non-exhaustive list of things to consider:
* the code is shared, but your understanding of what's prettier is subjective, so there's a mismatch right out of the gate here
* it increases the work needed to modify this code; folks have to now make a decision on every edit--change my variable names so the length matches, screw up the alignment and move on, or spend the extra few minutes realigning everything so it looks prettier.
* potentially impacts readability in several ways, such as requiring additional horizontal eyeball travel time, fighting the brain that naturally wants to treat the two <img width=50> terms as unrelated, creating longer horizontal scrollbars when zoomed in, etc.

Reply:
Allow me to speak from experience. I became the owner of a rather large file, 7000 LOC. The previous owner enjoyed lining things up vertically in this cute way. Whenever I make a change to this file, Visual Studio reformats it and destroys the vertical alignment, and my pull request says that I touched nearly half of the lines in the file. I have to go back and carefully record the lines that I intended to change, revert the file, check it back out, then carefully reapply the changes **via Notepad**, because I trust only Notepad to be able to modify a file without secretly reformatting it.

I do not enjoy working on that file.

Reply:
(GH needs the "caring" reaction.)

Reply:
@oldnewthing I can't reproduce that behavior, but I generally agree with the story's message here.